apps/judicial-system/web/src/routes/Shared/Login/Login.tsx

```diff
@@ -55,6 +71,22 @@ const Login = () => {
     }
   }
 
+  useEffect(() => {
+    if (user) {
+      const redirectRoute = isDefenceUser(user)
+        ? DEFENDER_CASES_ROUTE
+        : isPrisonStaffUser(user)
+        ? PRISON_CASES_ROUTE
+        : isCourtOfAppealsUser(user)
+        ? COURT_OF_APPEAL_CASES_ROUTE
+        : user.role === UserRole.ADMIN
+        ? USERS_ROUTE
+        : CASES_ROUTE
+
+      router.push(redirectRoute)
+    }
+  }, [router, user])
```

Comment:
I had it like that at first but I actually found it to be less readable because of the linting rules in our project which force the returns into separate lines. Also I prefer to always use {} in if statements even if they're in one line, it's safer to prevent unexpected occurrences if someone adds to the code later without adding them in